packages/panda-san/src/variables/form.scss

```diff
@@ -0,0 +1 @@
+$input-group-gap: 12px;
```

Comment:
do we need to convert this to rem?

Reply:
Since `mixins` depends on `variables` I believe we can't do that here. See [this issue](https://github.com/Lemonpeach/panda-san/issues/16). If we cannot than the solution is to wrap the variable in a `px-to-rem` call when we use it.

Reply:
Ah, got it.